docs/source/1.0/spec/aws/aws-restjson1-protocol.rst

```diff
@@ -312,10 +312,12 @@ Operation error serialization
 
 Error responses in the ``restJson1`` protocol are serialized identically to
 standard responses with one additional component to distinguish which error
-is contained. The component MUST be one of the following: an additional header
-with the name ``X-Amzn-Errortype``, a body field with the name ``code``, or a
-body field named ``__type``. The value of this component SHOULD contain only
-the :token:`shape name <smithy:identifier>` of the error's :ref:`shape-id`.
+is contained. New server-side protocol implementations MUST use a header field
+named ``X-Amzn-Errortype``. Clients MUST accept any one of the following: an
+additional header with the name ``X-Amzn-Errortype``, a body field with the
+name ``code``, or a body field named ``__type``. The value of this component
```

Comment:
nit: can we list `__type` before code? It seems like it would have higher precedence since it's much less likely to conflict with a modeled member named `code` that wasn't intended for this purpose.